xbmc/dialogs/GUIDialogPlayEject.cpp

```diff
@@ -75,38 +75,17 @@ void CGUIDialogPlayEject::OnInitWindow()
   CGUIDialogYesNo::OnInitWindow();
 }
 
-bool CGUIDialogPlayEject::ShowAndGetInput(const CFileItem & item,
-  unsigned int uiAutoCloseTime /* = 0 */)
+bool CGUIDialogPlayEject::ShowAndGetInput(const std::string strLine1,
+                                          const std::string strLine2,
+                                          unsigned int uiAutoCloseTime /* = 0 */)
 {
-  // Make sure we're actually dealing with a Disc Stub
-  if (!item.IsDiscStub())
-    return false;
 
   // Create the dialog
   CGUIDialogPlayEject * pDialog = (CGUIDialogPlayEject *)CServiceBroker::GetGUI()->GetWindowManager().
     GetWindow(WINDOW_DIALOG_PLAY_EJECT);
   if (!pDialog)
     return false;
 
-  // Figure out Lines 1 and 2 of the dialog
-  std::string strLine1, strLine2;
-  CXBMCTinyXML discStubXML;
-  if (discStubXML.LoadFile(item.GetPath()))
-  {
-    TiXmlElement * pRootElement = discStubXML.RootElement();
-    if (!pRootElement || StringUtils::CompareNoCase(pRootElement->Value(), "discstub") != 0)
-      CLog::Log(LOGERROR, "Error loading %s, no <discstub> node", item.GetPath().c_str());
```

Comment:
Header removal again. Doesn't appear to be any further clog usage

https://github.com/xbmc/xbmc/blob/a288c97e191a45ff3c5c3bb54c6ec0bebff854cb/xbmc/dialogs/GUIDialogPlayEject.cpp#L17

Reply:
removed this include